konan/konan.properties

```diff
@@ -400,14 +401,16 @@ targetSysRoot.linux_x64 = $gccToolchain.linux_x64/x86_64-unknown-linux-gnu/sysro
 # targetSysroot-relative.
 libGcc.linux_x64 = ../../lib/gcc/x86_64-unknown-linux-gnu/8.3.0
 targetCpu.linux_x64 = x86-64
-clangFlags.linux_x64 = -cc1 -target-cpu $targetCpu.linux_x64 -emit-obj -disable-llvm-optzns -x ir
+clangFlags.linux_x64 = -cc1 -target-cpu $targetCpu.linux_x64 -emit-obj -disable-llvm-optzns -x ir \
+   -ffunction-sections -fdata-sections
 clangNooptFlags.linux_x64 = -O1
-clangOptFlags.linux_x64 = -O3 -ffunction-sections
+clangOptFlags.linux_x64 = -O3
 clangDebugFlags.linux_x64 = -O0
 clangDynamicFlags.linux_x64 = -mrelocation-model pic
+clangStaticFlags.linux_x64 = -mrelocation-model pic
```

Comment:
Might be related: `OptimizationPipeline.kt` also has relocation mode. It is set to "default", and AFAIK it is not PIC on Linux.

Reply:
Yes, good catch. Looks like I'll have to add a separate properties for relocation mode.

Reply:
Done.